pandas/tests/groupby/test_function.py

```diff
@@ -1356,6 +1356,78 @@ def test_deprecate_numeric_only(
             method(*args, **kwargs)
 
 
+def test_deprecate_numeric_only_series(groupby_func, request):
+    # GH#46560
+    if groupby_func in ("backfill", "mad", "pad", "tshift"):
+        pytest.skip("method is deprecated")
+    elif groupby_func == "corrwith":
+        msg = "corrwith is not implemented on SeriesGroupBy"
+        request.node.add_marker(pytest.mark.xfail(reason=msg))
+
+    ser = Series(list("xyz"))
+    gb = ser.groupby([0, 0, 1])
+
+    if groupby_func == "corrwith":
+        args = (ser,)
+    elif groupby_func == "corr":
+        args = (ser,)
+    elif groupby_func == "cov":
+        args = (ser,)
+    elif groupby_func == "nth":
+        args = (0,)
+    elif groupby_func == "fillna":
+        args = (True,)
+    elif groupby_func == "take":
+        args = ([0],)
+    elif groupby_func == "quantile":
+        args = (0.5,)
+    else:
+        args = ()
+    method = getattr(gb, groupby_func, None)
+
+    try:
+        _ = method(*args)
```

Comment:
Just to confirm, all `groupby_func` not expected to fail so that's why `try/except` is used here and makes it harder to use `pytest.raises`?

Reply:
i.e. can this be tested like

```
if groupby_func in methods_ok:
    method(*args)
elif groupby_func in methods_raise_typeerror:
    with pytest.raises(TypeError, ...) as err:
        ...
elif groupby_func in methods_raise_valueerror:
    with pytest.raises(ValueError, ...) as err:
        ...
```

Reply:
Actually, none are successful. Some would be if the dtype was not object, but I plan to deprecate that in a follow up assuming no objection. Changed over to using pytest.raises.